app/Http/Controllers/Rpc/WalletController.php

```diff
@@ -65,31 +72,36 @@ public function calculateWithdrawal(Request $request)
 
     public function withdraw(Request $request)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1142)